src/Components/Components/src/BindElementAttribute.cs

```diff
@@ -38,6 +38,7 @@ public BindElementAttribute(string element, string suffix, string valueAttribute
             Element = element;
             ValueAttribute = valueAttribute;
             ChangeAttribute = changeAttribute;
+            Suffix = suffix;
```

Comment:
This feature must not be used much. Or tested 😄

Reply:
This should be nullable.